jvb/src/main/kotlin/org/jitsi/videobridge/relay/Relay.kt

```diff
@@ -108,6 +108,7 @@ class Relay @JvmOverloads constructor(
     /**
      * True if the ICE agent for this [Relay] will be initialized to serve as a controlling ICE agent, false otherwise.
      */
+    val meshId: String?,
```

Comment:
The `meshId` param is between `iceControlling` and its docs. 

Maybe add a note on the semantics of mesh ID somewhere (that it's associated with a bridge-to-bridge link and not a particular bridge)?